preprocessing/nextclade/src/loculus_preprocessing/backend.py

```diff
@@ -82,8 +82,7 @@ def parse_ndjson(ndjson_data: str) -> Sequence[UnprocessedEntry]:
             unalignedNucleotideSequences=json_object["data"]["unalignedNucleotideSequences"],
         )
         entry = UnprocessedEntry(
-            accessionVersion=f"{json_object['accession']}.{
-                json_object['version']}",
+            accessionVersion=json_object["accession"] + "." + json_object["version"],
```

Comment:
Looks like this change is the culprit - reverting
```suggestion
            accessionVersion=f"{json_object['accession']}.{
                json_object['version']}",
```